### ISSUE TITLE
Reuse-able Invoice

### ISSUE THREAD

**Issue by Otagera** [LaniJ/Invoice-Receipt-Generator#10]
Hi 👋,
I wonder if there would be a way to make the invoice reuse-able in the sense that one might have a reoccurring invoice to the same client, same product. So I was thinking instead of having to fill the form over and over again, is there a way to have that you can download the invoice in say a JSON format that you can re-upload and have it pre-fill the form and have your invoice.

There is an [issue](https://github.com/LaniJ/invoice-dragon/issues/1) that points to a standard for invoices but its kind of difficult for me to understand but I feel a simple JSON might work for this. 

If it is something that you'd like as a feature, I can start working on this based on an agreed JSON format. @LaniJ

**Comment by naumanzchaudhry** [comment]
@Otagera nice one. I was thinking a similar thing, Having a standardised format as a data source (json, csv etc) would definitely be a great addition

**Comment by Otagera** [comment]
Yes @naumanzchaudhry I am waiting for @LaniJ so we can try to get an agreed format and work something out
